report/finished/chapters/base_simulation.tex

```diff
@@ -0,0 +1,20 @@
+\subsection{Simulation}
+
+To verify the validity of the \textbf{PD} controller for this task, simulations were run in MATLAB. The simulation sets the motors to a common speed, so that the controller always makes the robot move forward when $\theta=0$ and lower the speed of a motor on one side to turn. This is the same way as the controller is implemented on the robot. Both motor speed can therefore be set into one variable that is defined as the difference between the motor speeds ($\Delta v = v_L-v_R$). \\
```

Comment:
were run in* might want to do past tense with "ran", however the internet is devided. 
sets = set

remove "that is" defined as....

Reply:
both motor speed -> both motor speeds

Reply:
Changed the motor speed and set. Will keep the present tense of run since the sentence is already in past tense